src/compiler/diagnosticMessages.json

```diff
@@ -1955,6 +1955,10 @@
         "category": "Error",
         "code": 2691
     },
+    "Cannot find name '{0}'. A type exists with this name, but no value.": {

```

Comment:
Ok, changed. I'm assuming TypeScript is a democracy and @RyanCavanaugh lost the popular vote :sweat_smile: